ground/src/main/res/layout/nav_drawer_header.xml

```diff
@@ -50,6 +54,57 @@
       style="@style/NavDrawerHeader.SecondaryText"
       android:text="@{user.email}"
       tools:text="User email" />
-
+    <com.google.android.material.divider.MaterialDivider
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content" />
+    <LinearLayout
+      style="@style/NavDrawerHeader"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="16dp"
+      android:fitsSystemWindows="true"
+      android:orientation="vertical">
+      <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/current_survey"
+        app:drawableStartCompat="@drawable/ic_content_paste"
+        tools:text="Current survey" />
+      <TextView
+        android:id="@+id/no_surveys_info"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/no_survey_selected"
+        android:visibility="gone"
+        />
+       <LinearLayout
+         android:id="@+id/survey_info"
+         android:layout_width="match_parent"
+         android:layout_height="wrap_content"
+         android:orientation="vertical">
+      <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:text="@{survey.title}"
+        android:textStyle="bold"
+        tools:text="Deforestation areas" />
```

Comment:
This seems out of place, as well as the tools text below: "A survey about farms."

Reply:
Changed it to more generic text, I was just using these to check out a semi-realistic preview of the layout in android studio